config/initializers/backtrace_silencers.rb

```diff
@@ -3,7 +3,8 @@
 # Be sure to restart your server when you modify this file.
 
 # You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
-# Rails.backtrace_cleaner.add_silencer { |line| line =~ /my_noisy_library/ }
+# Rails.backtrace_cleaner.add_silencer { |line| /my_noisy_library/.match?(line) }
 
-# You can also remove all the silencers if you're trying to debug a problem that might stem from framework code.
-# Rails.backtrace_cleaner.remove_silencers!
+# You can also remove all the silencers if you're trying to debug a problem that might stem from framework code
+# by setting BACKTRACE=1 before calling your invocation, like "BACKTRACE=1 ./bin/rails runner 'MyClass.perform'".
+Rails.backtrace_cleaner.remove_silencers! if ENV["BACKTRACE"]
```

Comment:
Instead of `BACKTRACE`, make it a bit more descriptive by saying `ENABLE_BACKTRACE`

Reply:
@unnitallman This was added by Rails itself. I kept it as the default. All the changes made to `config/**/*.rb` is done by Rails itself.

Is it a good idea to change the name given that with next upgrade Rails will try to re-add this? 

Let me know your thoughts.

Reply:
If its something that was auto-generated, let us keep that way.